### PR TITLE
Adding Readme Foundation

### PR DESCRIPTION
**Pull request by Laweeza**
**Fleshed out an initial README structure so that we have an outline to go off of later**

Roughly followed the guidelines in GLearn & created the following sections

- [x] Catchy headline / cool graphic (mostly the graphic part)

- [x] Who are the contributors (clickable profile images) 

- [x] Installation & Getting Started  (starting up the app)

- [x] Tech Stack (clickable hyperlinks) 

- [x] Created sections for About the App (Intro), How it Works (demo section), Challenges, Research, and Resources.

***Other Changes***
* Created initial mock data for my errand component + moved my data into the dummyData folder that Justin created under client/common. (Not entirely complete yet and will be done in another branch)
* Added the bagel logo as a png file into our assets folder
* Included .env* in .gitignore
* Added MIT license for kicks + generated a license badge from shields.io

**Please view & interact with the visual changes in the README branch if desired**
https://github.com/rpp30-boc-heartbleed/whimsical/tree/README

### CODE REVIEW

**Comment by Laweeza** [comment]
[Initial ReadME Setup](https://trello.com/c/DMgcWiT0/27-initial-readme-setup)

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 node_modules/
+.env*
 .expo/
 dist/
```
Comment (Laweeza) [inline .gitignore:4]: Included .env* in .gitignore.

File: README.md [hunk 2]
```diff
@@ -1 +1,217 @@
...
+</div>
+
+<h1 align="center"> Quick Bagel </h1>
+<p align="center">
+"This guy <b>keeps</b> taking my bagels." <i> - Tom from Myspace </i>
+<br/>
+A mobile application for you to <b>streamline</b> errands and pay good deeds forward!
+</p>
+
+<!-- PROJECT SHIELDS -->
```
Comment (Laweeza) [inline README.md]: Used a test shield for now. *MIT license* from https://shields.io/

File: README.md [hunk 3]
```diff
@@ -1 +1,217 @@
-This guy keeps taking my bagels
\ No newline at end of file
+<!-- PROJECT LOGO -->
+<div align="center">
+<img src="client/assets/mini-bag.png" alt="Bagel Logo" width="250" height="250">
+</div>
+
```
Comment (Laweeza) [inline README.md:5]: Bagel logo is a bit off center, will fix in near future.

File: README.md [hunk 4]
```diff
@@ -1 +1,217 @@
...
+<br/>
+A mobile application for you to <b>streamline</b> errands and pay good deeds forward!
+</p>
+
+<!-- PROJECT SHIELDS -->
+![license](https://img.shields.io/npm/l/nodemon)
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary> Table of Contents </summary>
```
Comment (Laweeza) [inline README.md:28]: Used Github Markdown to create a Table of Contents drop down menu.

https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/

File: README.md [hunk 5]
```diff
@@ -1 +1,217 @@
...
+<a href='https://www.mongodb.com/'><img src="https://cacm.acm.org/system/assets/0002/7119/042117_Theodo_MongoDB.large.jpg?1492791427&1492791427" width="70" height="70">
+</a>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+## How it Works
+
+<!-- Insert what app looks like visually, diagrams, demo video-->
+
```
Comment (Laweeza) [inline README.md:156]: Placeholder Markdown Table

https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables

File: README.md [hunk 6]
```diff
@@ -1 +1,217 @@
...
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Features
+- [ ] Placeholder Feature
+- [ ] Placeholder Feature
+- [x] Feature Implemented
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+## Team
```
Comment (Laweeza) [inline README.md:186]: We can try to change the square images into circles if we decide to keep the images in the Team section - in the near future.

File: README.md [hunk 7]
```diff
@@ -1 +1,217 @@
...
+</a>
+<a href="https://github.com/surekhaw">
+  <img src="https://github.com/surekhaw.png?size=50">
+</a>
+<a href="https://github.com/Laweeza">
+  <img src="https://github.com/Laweeza.png?size=50">
+</a>
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
```
Comment (Laweeza) [inline README.md:246]: Don't think this section is completely necessary, but left it in for now.

File: client/common/dummyData/mockErrandData.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
+// const userLocation = {
```
Comment (Laweeza) [inline client/common/dummyData/mockErrandData.js:1]: Still working on fleshing out mockErrandData. Will complete in a separate PR.

File: server/controllers/errandTracker.js [hunk 9]
```diff
@@ -1,4 +1,4 @@
-const { Example } = require('../models/errandTracker');
+const { Example } = require('../models/errand');
```
Comment (Laweeza) [inline server/controllers/errandTracker.js:1]: I just changed the file name from errandTracker to errand lol

File: server/models/errand.js [hunk 10]
```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
```
Comment (Laweeza) [inline server/models/errand.js:2]: My initial schema, will iterate upon it slowly.

**Review by surekhaw — APPROVED** [review]
this looks amazing!

**Pull request merged**